qemu/tests/rh_kernel_update.py

```diff
@@ -330,6 +329,13 @@ def upgrade_guest_pkgs(session, pkg, debuginfo=False,
             if params.get("install_rpm_from_local") == "yes":
                 rpm_install_func = copy_and_install_rpm
 
+            kernel_deps_pkgs = params.get("kernel_deps_pkgs").split()
+            if kernel_deps_pkgs:
+                for pkg in kernel_deps_pkgs:
+                    arch = params.get("arch_dracut",
```

Comment:
We can use params.object_params(pkg).get(arch) to get arch param for each packages, not limited your code to dracut package only.